### PR TITLE
migrate connAck from SendEvent to CreateParticipantConnection

### PR DESCRIPTION
**Pull request by spencerlepine**
*Issue #, if available:*

*Description of changes:*
Update ChatController to call CreateParticipantConnection by default, instead of SendEvent. Removed fallback logic, and added test coverage for expected default call behavior. Documentation: [API_CreateParticipantConnection](https://docs.aws.amazon.com/connect-participant/latest/APIReference/API_CreateParticipantConnection.html)

- [x] Migrate connAck from SendEvent to CreateParticipantConnection
- [x] Resolve[ dependabot alert #19](https://github.com/amazon-connect/amazon-connect-chatjs/security/dependabot/19)
- [x] Update CHANGELOG + include PR #142
- [x] Minor bump to `v1.4.0`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "amazon-connect-chatjs",
```
Comment (mhiaror) [inline package.json:2]: Instead of a patch release , lets do a minor release ?
1.4.0

Comment (spencerlepine) [reply package.json:2]: Good catch, will publish rev now.

**Review by mhiaror — APPROVED** [review]

**Comment by mhiaror** [comment]
Always run a test by using the `dist/` package to launch `amazon-connect-chat-interface` manually to catch any Customer impacting issue before release.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.4.0]
+### Added
```
[the remark this answers is not in the document]
Comment (spencerlepine) [reply CHANGELOG.md:10]: Rewording in next revision 👍

**Review by haomingli2020 — APPROVED** [review]

**Pull request merged**